packages/gatsby/src/utils/prepare-regex.ts

```diff
@@ -2,15 +2,6 @@ import _ from "lodash"
 
 export const prepareRegex = (str: string): RegExp => {
   const exploded = str.split(`/`)
-  const regex = new RegExp(
-    exploded
-      .slice(1, -1)
-      .join(`/`)
-      // Double escaping is needed to get past the GraphQL parser,
-      // but single escaping is needed for the RegExp constructor,
-      // i.e. `"\\\\w+"` for `/\w+/`.
-      .replace(/\\\\/, `\\`),
```

Comment:
IIRC this was the actual problem, this regex was missing a global flag?

But I mean if we can drop this entirely that's even better. Will ask @vladar to take a sanity check for graphql.

Reply:
Double escaping my regex stops the errors, but doesn't return any results the same way the graphiql explorer does... Completely out of ideas.